wsd/FileServer.cpp

```diff
@@ -544,7 +543,7 @@ void FileServerRequestHandler::handleRequest(const HTTPRequest& request,
 #if ENABLE_DEBUG
         if (Util::startsWith(relPath, std::string("/wopi/files"))) {
             handleWopiRequest(request, requestDetails, message, socket);
-            return;
+            return ResourceAccessDetails();
```

Comment:
Not a huge fan of having this as a return value and all these un-related code-paths having this return.
Can we either split the function, or have an optional std::function callback that is usually null for the one path that matters, or pass a reference to an empty shared_ptr in for the path that matters or somesuch ? =)

Reply:
Agreed. Reworked it.

Reply:
Didn't see that in this patch; but look forward to it =)

Reply:
Yes, was in the update that wasn't yet uploaded at the time. Now updated.